src/main/resources/scripts/windowsInstallGitScript.ps1

```diff
@@ -4,7 +4,7 @@ $latestRelease = Invoke-WebRequest -UseBasicParsing $source -Headers @{"Accept"=
 $json = $latestRelease.Content | ConvertFrom-Json
 $latestVersion = $json.tag_name
 $versionHead = $latestVersion.Substring(1, $latestVersion.IndexOf("windows")-2)
-$source = "https://github.com/git-for-windows/git/releases/download/v${versionHead}.windows.1/Git-${versionHead}-64-bit.exe"
+$source = "https://github.com/git-for-windows/git/releases/download/v${versionHead}.windows.2/Git-${versionHead}.1-64-bit.exe"
```

Comment:
Shouldn‘t the filename also end with `.2-64-bit.exe`?

Reply:
Yes. pushing the update.  My bad.